static/redfish/v1/schema/OemComputerSystem_v1.xml

```diff
@@ -23,6 +23,28 @@
                 <Annotation Term="OData.Description" String="OemComputerSystem Oem properties." />
                 <Annotation Term="OData.AutoExpand"/>
                 <Property Name="OpenBmc" Type="OemComputerSystem.OpenBmc"/>
+                <Property Name="IBM" Type="OemComputerSystem.IBM"/>
+            </ComplexType>
+
+            <ComplexType Name="IBM" BaseType="Resource.OemObject">
+                <Annotation Term="OData.AdditionalProperties" Bool="true" />
+                <Annotation Term="OData.Description" String="Oem properties for IBM." />
+                <Annotation Term="OData.AutoExpand"/>
+                <Property Name="LampTest" Type="OemComputerSystem.IBM">
```

Comment:
Type would need to be `"Edm.Boolean"`.
Please look at https://github.com/ibm-openbmc/bmcweb/pull/798